ui/src/app/components/rename-modal.spec.tsx

```diff
@@ -60,3 +63,72 @@ describe('should show error if new name already exist', () => {
     }
   );
 });
+
+describe('RenameModal', () => {
```

Comment:
nit: we already have a section with this heading.  Also, we prefer `RenameModal.name` because that will get updated if we rename the component in the future.